openvino/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+# ******************************************************************************
+# Copyright 2017-2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ******************************************************************************
+
+cmake_minimum_required(VERSION 3.10)
+
+file(REMOVE "${CMAKE_BINARY_DIR}/openvino_targets_developer.cmake")
+
+unset(OpenVINODeveloperPackageTargets CACHE)
+
+function(openvino_developer_export_targets)
+    set(OpenVINODeveloperPackageTargets "${OpenVINODeveloperPackageTargets};${ARGV}")
+
+    # to allow exporting of aliased targets with the original names
+    foreach(target_name ${OpenVINODeveloperPackageTargets})
+        if(TARGET "${target_name}")
+            get_target_property(original_name ${target_name} ALIASED_TARGET)
+            if(TARGET "${original_name}")
+                message(STATUS "The name ${target_name} is an ALIAS for ${original_name}. "
+                        "It will be exported to the InferenceEngineDeveloperPackage with the original name.")
+                list(REMOVE_ITEM OpenVINODeveloperPackageTargets ${target_name})
+                list(APPEND OpenVINODeveloperPackageTargets ${original_name})
+            endif()
+        endif()
+    endforeach()
+
+    list(REMOVE_DUPLICATES OpenVINODeveloperPackageTargets)
+    set(OpenVINODeveloperPackageTargets "${OpenVINODeveloperPackageTargets}" CACHE INTERNAL
+        "Paths to extra Inference Engine plugins" FORCE)
+endfunction()
+
+function(openvino_developer_export)
+    export(TARGETS ${OpenVINODeveloperPackageTargets} NAMESPACE openvino::
+            APPEND FILE "${CMAKE_BINARY_DIR}/openvino_targets_developer.cmake")
+
+    # Custom target to build only Openvino Developer Package targets
+    add_custom_target(openvino_dev_targets ALL DEPENDS ${OpenVINODeveloperPackageTargets})
```

Comment:
looks like we don't need `openvino_developer_export` as all since OpenVINO targets are exported as `IE::` anyway.

Reply:
I think, it's wrong to use internal cmake functions to export common libraries. Here some refactoring is nessesary. All libraries is better to export in top level CmakeLists.txt file in openvino:: namespace.